screens/TabFiveScreen.tsx

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+
+import EditScreenInfo from '../components/EditScreenInfo';
+import { Text, View } from '../components/Themed';
+
+export default function TabThreeScreen() {
```

Comment:
Don't forget to change this to be `TabFiveScreen`!

Reply:
Ooh yes just fixed!